### PR TITLE
protection from massive powheg weight

### PR DESCRIPTION
**Pull request by gfacini**

### CODE REVIEW

**Comment by gfacini** [comment]
@kratsg @mmilesi 
There is a huge weight in this sample that needs to be removed like it never existed.
Any strong objections?

**Comment by kratsg** [comment]
Why does it have a huge weight in the first place? Do we really want to hardcode that? That seems out of the context of xAH :(

**Comment by gfacini** [comment]
Generator flaw.  It is not analysis specific - no one wants this event.  See slide 2:
http://hep.uchicago.edu/~johnda/work/PowhegVsPythia.pdf

**Comment by kratsg** [comment]
Can we detect the sample being used automatically from the dataset ID and remove it, rather than a configurable variable?

File: Root/BasicEventSelection.cxx [hunk 1]
```diff
@@ -374,6 +377,12 @@ EL::StatusCode BasicEventSelection :: initialize ()
     }
   }
 
+  m_cleanPowheg = false;
```
Comment (gfacini) [inline Root/BasicEventSelection.cxx]: @kratsg

**Comment by gfacini** [comment]
L380!
If the runnumber matches then the check is done

**Comment by kratsg** [comment]
The runNumber is unique within a sample, but not within multiple samples. I'm saying, we should have the script grab the DID automatically and apply the filter if the DID matches. Rather than having a configurable check. Not everyone will know to configure it.

**Comment by gfacini** [comment]
runNumber is DID!

**Comment by kratsg** [comment]
I'm stupid. This looks fine. @johnda102 will be pleased.

**Comment by gfacini** [comment]
1) False
2) Who cares!? =p

**Pull request merged**